website/src/components/Layout/Header/index.tsx

```diff
@@ -31,6 +31,9 @@ export const Header: React.FC = () => {
             <ul className="dropdown-menu bg-light">
               <Link to="/projects" className="nav-link navigation-link text-center text-secondary">Projects</Link>
               <Link to="/publications" className="nav-link navigation-link text-center text-secondary">Publications</Link>
+              <li className="mx-5 mx-md-4 nav-item">
```

Comment:
Why did you need this <li> with those classes?
The result is different for this link than for projects and publications

Reply:
I deleted it

Reply:
Why did you delete it? We still need to have a trap link in the header. It's just the css classes that were incorrect

Reply:
Some classes are still missing on the trap link